### PR TITLE
Improve implementers agreement regarding composite components

### PR DESCRIPTION
**Pull request by jasollien**

### CODE REVIEW

**Comment by ykulbak** [comment]
Discussed a best practice that selection should also be clarified that when a root element is `selected` in BCF then
- The geometry of the children should be highlighted
- Only the root element should appear as selected in the tree

**Comment by cschladetsch** [comment]
Should a different color/representation be used to indicate between children of a `selected` node and other explicitly selected children?

**Comment by pasi-paasiala** [comment]
> Should a different color/representation be used to indicate between children of a `selected` node and other explicitly selected children/nodes?

From end users' perspective there's no difference. This mainly an optimization to keep the BCF message sizes small. If the user has had, say a curtain wall, fully selected, then by passing the parent node in BCF, we're just passing the same information in a compact way.

File: Documentation/README.md [hunk 1]
```diff
@@ -245,13 +243,33 @@ The visibility is applied in following order:
...
+
+##### When creating the viewpoint
+
+* If the container element and all its parts have the same color then only the container element should be added to the component list
+* If only some parts have the same color, then only those parts should be added to the component list.
+* Adding all the parts to the component list is allowed and would result in excluding any parts that will be added in later revisions of the model
+
+##### When visualizing the viewpoint
+
+The color of a container element should be applied to all its parts.
```
Comment (jasollien) [inline Documentation/README.md:272]: This text feels a bit made for "coloring" only.

Comment (jasollien) [inline Documentation/README.md:272]: This text feels a bit made for "coloring" only.

Comment (ykulbak) [reply Documentation/README.md:272]: It says on line 262 "using coloring as an example". I think it's easier to understand when it's specific, do you agree?

**Review by ykulbak — APPROVED** [review]

**Pull request merged**